src/shared/modules/state-and-params.ts

```diff
@@ -18,8 +19,12 @@ import {
     getMapItemsIgnores,
     mergeParamsWithAliases,
     getCurrentVersion,
+    pickActionParamsFromParams,
+    hasActionParam,
+    pickExceptActionParamsFromParams,
+    transformParamsToActionParams,
 } from './helpers';
-import {META_KEY} from '../constants';
+import {intersection, omit} from 'lodash';
```

Comment:
It is better to use: `import A from 'lodash/A'`

Reply:
fixed